pylint/test/functional/singledispatch_functions.rc

```diff
@@ -0,0 +1,3 @@
+[testoptions]
+;http://bugs.python.org/issue10445
+except_versions=3.2,3.3
```

Comment:
Can't we use minver for this, with two separate files for Python 2 and Python 3?